es/src/main/java/io/apiman/distro/es/Bootstrapper.java

```diff
@@ -76,23 +60,33 @@ public void contextInitialized(ServletContextEvent sce) {
             URL esDistroUrl = resolveEsDistro(sce);
             File tmpDir = new File(getTempDir());
 
-            Builder builder = ApimanEmbeddedElastic.builder()
-                .withPort(config.getHttpPortRange())
-                .withDownloadUrl(esDistroUrl)
-                .withDownloadDirectory(tmpDir)
-                .withInstallationDirectory(tmpDir)
-                .withSetting("path.data", esHome)
-                .withCleanInstallationDirectoryOnStop(false)
-                .withSetting(PopularProperties.TRANSPORT_TCP_PORT, config.getTransportPortRange())
-                .withSetting(PopularProperties.CLUSTER_NAME, clusterName)
-                .withStartTimeout(1, TimeUnit.MINUTES);
-
-            if (config.getBindHost() != null) {
-                builder.withSetting("network.bind_host", config.getBindHost());
-            }
-
-            node = builder.build().start();
-        } catch (IOException | InterruptedException e) {
+            node = new FixedPortEsContainer(
+                 "docker.elastic.co/elasticsearch/elasticsearch-oss:" + EsConstants.getEsVersion())
+                 .withFixedExposedPortRange(config.getHttpPortRange())
+                 .withStartupTimeout(Duration.ofHours(1))
+                 .waitingFor(Wait.forHttp("/"));
+
+            System.out.println("If you have never downloaded elasticsearch before, this may take a few moments...");
+
+            node.start();
+//
+//            Builder builder = ApimanEmbeddedElastic.builder()
```

Comment:
Whoops, remove this, likely not needed. Do we need an equivalent for bind host with docker or not 🤔?